pinc/ProjectSearchResults.inc

```diff
@@ -717,23 +717,23 @@ function get_search_configure_link()
     return "<a href='?show=config&amp;origin=$origin'>" . _("Configure Result") . "</a>";
 }
 
-function make_url($new_array, $anchor = '')
+function make_search_page_url(array $new_array, string $anchor = ''): string
 {
     return "?" . http_build_query(array_replace($_GET, $new_array)) . "$anchor";
 }
 
-function make_link($new_array, $text, $anchor = '')
+function make_search_page_link(array $new_array, string $text, string $anchor = ''): string
```

Comment:
Ditto about narrowing the type of `$new_array`:

```suggestion
/** @param array<string, int> $new_array */
function make_search_page_link(array $new_array, string $text, string $anchor = ''): string
```

Reply:
They both need to be `/** @param array<string, mixed> $new_array */` -- I've added a new commit.